tests/lax_numpy_test.py

```diff
@@ -1073,6 +1073,20 @@ def testIn1d(self, element_shape, test_shape, dtype, invert):
     self._CheckAgainstNumpy(np_fun, jnp_fun, args_maker)
     self._CompileAndCheck(jnp_fun, args_maker)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_{}_{}".format(
+       jtu.format_shape_dtype_string(shape1, dtype),
+       jtu.format_shape_dtype_string(shape2, dtype)),
+       "shape1": shape1, "shape2": shape2, "dtype": dtype}
+      for dtype in default_dtypes
```

Comment:
Is it worth testing cases when the arrays have different types?

Reply:
Done

Reply:
Done